### PR TITLE
Create staging environment, separate from prod environment.

### PR DESCRIPTION
**Pull request by ndmckinley**
### Description
Change names to avoid conflicts, swap the URLs around a bit.  :)

Note: the staging logins aren't going to work yet - I have meetings upcoming to get those working.

### Checklist
- [ ] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### CODE REVIEW

File: bin/check-cfn-change [hunk 1]
```diff
@@ -12,27 +12,27 @@ TIMESTAMP=$(date +%s)
 REGION=us-west-2
 
 echo "checking if a changeset exists already..."
-if aws cloudformation describe-change-set --region ${REGION}  --change-set-name auto-changeset-pr-$PR_NUMBER  --stack-name civiform > /dev/null; then
+if aws cloudformation describe-change-set --region ${REGION}  --change-set-name auto-changeset-pr-$PR_NUMBER  --stack-name civiform-staging > /dev/null; then
```
Comment (hmhsu) [inline bin/check-cfn-change:15]: Thought(non-blocking): mind making this a variable? we might be able to reuse this script to detect changes to different environments. Feel free to ignore if not useful.

File: bin/deploy-prod [hunk 2]
```diff
@@ -13,4 +13,4 @@ docker push public.ecr.aws/t1q6b4h2/universal-application-tool:latest
 TIMESTAMP=$(date +%s)
 
 aws s3 sync ./infra s3://seattle-civiform-cftmpl/${TIMESTAMP}
-aws cloudformation update-stack --region ${REGION} --stack-name civiform --template-url  https://seattle-civiform-cftmpl.s3-${REGION}.amazonaws.com/$TIMESTAMP/stack.yaml --parameters "[{\"ParameterKey\": \"Timestamp\", \"ParameterValue\": \"$TIMESTAMP\"}]"
+aws cloudformation update-stack --region ${REGION} --stack-name civiform --template-url  https://seattle-civiform-cftmpl.s3-${REGION}.amazonaws.com/$TIMESTAMP/stack.yaml --parameters "[{\"ParameterKey\": \"Timestamp\", \"ParameterValue\": \"$TIMESTAMP\"}, {\"ParameterKey\": \"Environment\", \"ParameterValue\": \"prod\"}]"
```
Comment (hmhsu) [inline bin/deploy-prod:16]: Thoughts (non-blocking): Similarly we might be able to use the same script.

**Review by hmhsu — APPROVED** [review]

File: bin/check-cfn-change (hunk 1 above)
Comment (ndmckinley) [reply bin/check-cfn-change:15]: Done, good idea.

File: bin/deploy-prod (hunk 2 above)
Comment (ndmckinley) [reply bin/deploy-prod:16]: They're going to need to build the container a little differently once I get around to that, so leaving them separate.

File: bin/check-cfn-change [hunk 3]
```diff
@@ -10,29 +11,39 @@ set -e
...
 	echo "deleting..."
-	aws cloudformation delete-change-set --region ${REGION}  --change-set-name auto-changeset-pr-$PR_NUMBER  --stack-name civiform-staging
+	aws cloudformation delete-change-set --region ${REGION}  --change-set-name auto-changeset-pr-$PR_NUMBER  --stack-name "$STACK"
 	sleep 5
 fi
 
 aws s3 sync ./infra s3://seattle-civiform-cftmpl/${TIMESTAMP}
 
-aws cloudformation create-change-set --region ${REGION} --include-nested-stacks --stack-name civiform-staging --change-set-name auto-changeset-pr-$PR_NUMBER --template-url  https://seattle-civiform-cftmpl.s3-${REGION}.amazonaws.com/$TIMESTAMP/stack.yaml --parameters "[{\"ParameterKey\": \"Timestamp\", \"ParameterValue\": \"$TIMESTAMP\"}, {\"ParameterKey\": \"Environment\", \"ParameterValue\": \"staging\"}]"
+aws cloudformation create-change-set --region ${REGION} --include-nested-stacks --stack-name "$STACK" --change-set-name auto-changeset-pr-$PR_NUMBER --template-url  https://seattle-civiform-cftmpl.s3-${REGION}.amazonaws.com/$TIMESTAMP/stack.yaml --parameters "[{\"ParameterKey\": \"Timestamp\", \"ParameterValue\": \"$TIMESTAMP\"}, {\"ParameterKey\": \"Environment\", \"ParameterValue\": \"staging\"}]"
```
Comment (hmhsu) [inline bin/check-cfn-change:34]: You might want to replace `staging` with `"$ENVIRONMENT"`

**Pull request merged**